testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeChangeDataFeedCompatibility.java

```diff
@@ -452,17 +451,17 @@ public void testMergeDeleteIntoTableWithCdfEnabled(String columnMappingMode)
         }
     }
 
-    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS}, dataProvider = "columnMappingModeDataProvider")
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
     @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
-    public void testMergeMixedDeleteAndUpdateIntoTableWithCdfEnabled(String columnMappingMode)
+    public void testMergeMixedDeleteAndUpdateIntoTableWithCdfEnabled()
```

Comment:
I'd leave this test as it is,I dion't see this case being covered anywhere else

Reply:
Do you think `TestDeltaLakeConnectorTest#testReadMergeChanges` is insufficient?

Reply:
oh i forgot about that one, sure it is fine